README.md

```diff
@@ -59,33 +59,46 @@ poetry build-project --with-top-namespace my_namespace
 
 #### The build output
 
-Default behaviour of `build-project` (i.e. without any custom top namespace flag):
+###### Default(no flag)
 ```shell
 /my_package
    __init__.py
    my_module.py
 ```
 
-By using the `--with-top-namespace` flag, the built artifact will look something like this:
+###### Namespace(`--with-top-namespace=my_namespace`)
 ```shell
 my_namespace/
     /my_package
        __init__.py
        my_module.py
 ```
 
+###### Namespace with path(`--with-top-namespace=my_namespace/subdir`)
+```shell
+my_namespace/
+    /my_package
```

Comment:
I think this should be:
/subdir
    /my_package

right?

Reply:
grrr, yes. good find.